### PR TITLE
Hosted article - multiple inline videos tracking fixed

### PR DESCRIPTION
**Pull request by Calanthe**
## What does this change?

Fix for the multiple inline videos tracking issue (only the latest one was tracked correctly).
## What is the value of this and can you measure success?

We can track all of the videos now.
## Does this affect other platforms - Amp, Apps, etc?

no

## Screenshots

screen shot 2016-09-28 at 10 24 41
## Request for comment

@lps88 @akash1810

### CODE REVIEW

**Comment by lps88** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/hosted/video.js [hunk 1]
```diff
@@ -116,7 +116,8 @@ define([
 
                         player.ready(function () {
                             var vol;
-                            duration = parseInt(this.duration(), 10);
+                            var player = this;
```
Comment (akash1810) [inline static/src/javascripts/projects/commercial/modules/hosted/video.js]: I don't think this variable is needed as we're assigning it [above](https://github.com/guardian/frontend/pull/14486/files#diff-547d3fcebab67a2a4a843b1d1c287f52R113).

Comment (Calanthe) [inline static/src/javascripts/projects/commercial/modules/hosted/video.js]: It is needed here, this is what was broken. In this scope, `player` points to the wrong object (always the last video), so `this` is the correct reference.

**Review by akash1810 — APPROVED** [review]
👍

**Pull request merged**